<def f='include/boost/thread/exceptions.hpp' l='182' type='void boost::invalid_thread_argument::invalid_thread_argument(int ev)'/>
